riscv-rt/Cargo.toml

```diff
@@ -24,6 +24,7 @@ riscv-target-parser = { path = "../riscv-target-parser", version = "0.1.0" }
 
 [dependencies]
 riscv = { path = "../riscv", version = "0.12.0" }
+riscv-macros = { path = "../riscv/macros", version = "0.1.0" }
```

Comment:
```suggestion
riscv = { path = "../riscv", version = "0.12.0", features = ["riscv-macros"] }
```

Reply:
That's actually a default feature, so I just removed the dependency.